Code/Editor/Core/EditorActionsHandler.h

```diff
@@ -60,6 +61,11 @@ class EditorActionsHandler
     void AfterEntitySelectionChanged(
         const AzToolsFramework::EntityIdList& newlySelectedEntities, const AzToolsFramework::EntityIdList& newlyDeselectedEntities);
 
+    // Recent Files
+    bool IsRecentFileActionActive(int index);
```

Comment:
Nitpick: Could make this `const`

Reply:
There's always one lol. I'll add this in the follow up, thanks :D